### PR TITLE
DLS-8423 added in claim for exports and how many for correct return a…

### PR DESCRIPTION
**Pull request by ChrisHillCap**
…nd some accesibility fixes
image
image

### CODE REVIEW

**Comment by ChrisHillCap** [comment]
https://github.com/hmrc/soft-drinks-industry-levy-ui-tests/pull/138

File: app/views/correctReturn/ClaimCreditsForExportsView.scala.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("correctReturn.claimCreditsForExports.title"))) {
+
+    @formHelper(action = routes.ClaimCreditsForExportsController.onSubmit(mode), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+            }
```
Comment (drewmiley) [inline app/views/correctReturn/ClaimCreditsForExportsView.scala.html:38]: Indentation?

File: app/views/correctReturn/ClaimCreditsForExportsView.scala.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            field = form("value"),
+            legend = LegendViewModel(messages("correctReturn.claimCreditsForExports.title")).withCssClass("govuk-fieldset__legend--m"),
+            hint = Some(Hint(content = messages("correctReturn.claimCreditsForExports.subHeading")))
+                )
+            )
+
+        @exportedWhatCanIClaimCreditFor(form)(messages)
+        @govukButton(
+            ButtonViewModel(messages("site.saveContinue"))
+            )
```
Comment (drewmiley) [inline app/views/correctReturn/ClaimCreditsForExportsView.scala.html:58]: Indentation?

File: app/views/correctReturn/HowManyClaimCreditsForExportsView.scala.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        }
+
+        <h1 class="govuk-heading-m">@messages("correctReturn.howManyCreditsForExport.title")</h1>
+        <p class="govuk-body">@messages("correctReturn.howManyCreditsForExport.subtext")</p>
+
+        @lowAndHighBandInputs(form)(messages)
+        @exportedWhatCanIClaimCreditFor(form)(messages)
+        @govukButton(
+            ButtonViewModel(messages("site.saveContinue"))
+            )
```
Comment (drewmiley) [inline app/views/correctReturn/HowManyClaimCreditsForExportsView.scala.html:50]: Indentation?

File: test/views/correctReturn/ClaimCreditsForExportsViewSpec.scala [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    "should contain the expected title" in {
+      document.title() mustBe "Do you want to claim a credit for liable drinks that have been exported? - Soft Drinks Industry Levy - GOV.UK"
+    }
+
+    "should include the expected hint text" in {
+      val hint = document.getElementById("value-hint")
+      hint.className() mustBe "govuk-hint"
+      hint.text() mustBe "You can only claim a levy credit for drinks that you have paid the levy on or will pay the levy on. Do not include drinks produced for small producers or imported from them."
+    }
+    "should include a legend with the expected heading (the oldest year within the returns)" in {
```
Comment (drewmiley) [inline test/views/correctReturn/ClaimCreditsForExportsViewSpec.scala:59]: Not sure this test name is correct

File: app/views/helpers/exportedWhatCanIClaimCreditFor.scala.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(govukDetails: GovukDetails)
+
+@(form: Form[_])(implicit messages: Messages)
+
+@htmlContent = {
```
[the remark this answers is not in the document]
Comment (ChrisHillCap) [reply app/views/helpers/exportedWhatCanIClaimCreditFor.scala.html:21]: different content

**Review by drewmiley — DISMISSED** [review]

**Review by drewmiley — APPROVED** [review]

**Pull request merged**